src/rules/use-tokens.js

```diff
@@ -152,6 +152,11 @@ const checkColor = (node, context) => {
         ];
       },
     });
+  } else {
+    context.report({
+      node,
+      message: `Unknown colour, use a Backpack token instead`,
```

Comment:
It pains me to say this but we should use the US spelling of colour as it's technically code.

How about:

`Colors should use Backpack tokens`